### PR TITLE
Resto druid mana used

### PR DESCRIPTION
**Pull request by fel1n3**
Added mana used doughnut chart to resto druid. Code reused from Hunter focus used module. Also added Uldir boss icons from achievements (not headshots). Allows player to see where their mana is going.

### CODE REVIEW

File: src/CONTRIBUTORS.js [hunk 1]
```diff
@@ -457,3 +457,14 @@ export const Khadaj = {
     link: 'https://worldofwarcraft.com/en-us/character/firetree/khadaj',
   }],
 };
+export const fel1ne = {
+  nickname: 'fel1ne',
+  github: 'fel1n3',
+  discord: 'Dr. fel1ne#5614',
+  avatar: require('./images/fel1ne-avatar.png'),
```
Comment (jjs451) [inline src/CONTRIBUTORS.js]: You need to add your avatar to version control to Git otherwise the server won't have it

**Comment by jjs451** [comment]
Could you attach a screenshot of both the mana chart and the icons?  Screenshots usually help us approve PRs faster.  Also, please remove package-lock.json from your commits

**Comment by MartijnHols** [comment]
image

**Comment by fel1n3** [comment]
Wew thank you! <3

**Pull request merged**